src/WebJobs.Extensions.DocumentDB/Trigger/DocumentDBTriggerAttribute.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs
+{
+    using System;
+    using Microsoft.Azure.WebJobs.Extensions.DocumentDB;
+
+    /// <summary>
+    /// Defines the [DocumentDBTrigger] attribute
+    /// </summary>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1019:DefineAccessorsForAttributeArguments")]
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class DocumentDBTriggerAttribute : Attribute
+    {
+        /// <summary>
+        /// Triggers an event when changes occur on a monitored collection
+        /// </summary>
+        /// <param name="monitorCollectionDatabase">Name of the database of the collection to monitor for changes</param>
+        /// <param name="monitorCollection">Name of the collection to monitor for changes</param>
+        public DocumentDBTriggerAttribute(string monitorCollectionDatabase, string monitorCollection)
+        {
+            if (string.IsNullOrWhiteSpace(monitorCollection))
+            {
+                throw new ArgumentException("Missing information for the collection to monitor", "monitorCollection");
+            }
+
+            if (string.IsNullOrWhiteSpace(monitorCollectionDatabase))
+            {
+                throw new ArgumentException("Missing information for the collection to monitor", "monitorCollectionDatabase");
+            }
+
+            CollectionName = monitorCollection;
+            DatabaseName = monitorCollectionDatabase;
+            LeaseCollectionName = DocumentDBTriggerConstants.DefaultLeaseCollectionName;
```

Comment:
No, they have to be stored in a different collection always. And it's recommended that it's a different service if possible. That's the reason of the second connection string. Using it is the recommended scenario.